example/ExamplePage.js

```diff
@@ -14,7 +14,7 @@ require('brace/ext/language_tools');
 require('brace/mode/json');
 require('brace/theme/github');
 require('rc-select/assets/index.css');
-import RcSelect from 'react-schema-form-rc-select/lib/RcSelect';
+//import RcSelect from 'react-schema-form-rc-select/lib/RcSelect';
```

Comment:
What was the issue with RcSelect?

Reply:
no issues, this is one of really eary changes that I've done in the process of researching of library structure. I will fix soon too